README.md

```diff
@@ -48,100 +48,127 @@ the reader to fulfill an action.
+This section demonstrates the process for deploying Konflux locally, onboarding users,
 and building and releasing an application. The procedure contains two options for the
 user to choose from for onboarding applications to Konflux:
 
 - Using the Konflux UI
 - Using Kubernetes manifests
 
-Each of those options has its pros and cons: the procedure described using the UI,
-provides more streamlined user experience once setup is done, but it requires using
-[Quay.io](https://quay.io) for image registry and requires some additional initial setup
-steps comparing to using Kubernetes manifest alone. The latter also supports using any
-image registry.
+Each of these options has its pros and cons. The Konflux UI procedure provides a more
+streamlined user experience once you finish installing and configuring it. However,
+it requires using [Quay.io](https://quay.io) for image registry, plus some additional
+initial setup steps compared to using a Kubernetes manifest alone.
 
-**Note:** The procedure that is described using the UI can also be fulfilled using CLI
-and Kubernetes manifests.
+**Note:** You can configure your server to operate the same way using either the
+Konflux UI method or the terminal and Kubernetes manifests procedure. For example,
+you can also configure Konflux to support an image registry using the terminal process.
 
-In both cases, the recommended way to try out Konflux is using
-[Kind](https://kind.sigs.k8s.io/).
-The process below creates a Kind cluster using the provided config in this repository.
-The config tells Kind to forward port `9443` from the host to the Kind cluster. The port
-forwarding is needed for accessing Konflux.
+In both cases, the recommended way to try out Konflux is using [Kind](https://kind.sigs.k8s.io).
+This guide will walk you through creating a Kind cluster using the `kind-config.yaml`
+manifest, located in the root directory of this repository. This manifest tells
+[Kind](https://kind.sigs.k8s.io) to forward port `9443` from the host to the
+[Kind](https://kind.sigs.k8s.io) cluster. Kind must configure this port forwarding
+to access Konflux.
 
-**Note:** If using a remote machine for setup, you'd need to port-forward port `9443` on
-the remote machine to port `9443` on your local machine to be able to access the UI from
-your local machine.
+**Note**: For security reasons, Konflux limits browser access to the local server.
+If you are using a remote machine to set up Konflux, you'll need to use SSH to set
+up port forwarding to access Konflux through your local computer's browser. To do this,
+open an additional terminal and type the following command:
+
+```bash
+ssh -L 9443:localhost:9443 <remote_host_user_name>@<remote_host_ip_address>
+```
 
 ## Machine Minimum Requirements
 
-The deployment is currently only supported on **x86_64 Linux** platforms.
+Konflux currently only supports the **x86_64 Linux** platform.
```

Comment:
I'm not sure if this statement is true for any deployment of Konflux. The way Konflux is deployed in this context was only tested on x86_64 Linux platforms.

Reply:
You're right. I only heard of Konflux three weeks ago, so I merely adjusted the existing sentence for clarity. I'll add that it also supports arm64.